### PR TITLE
make att.labelled available for textelements

### PR DESCRIPTION
**Pull request by rettinghaus**
This PR makes `@label` available for all text elements and adds corresponding labels to the parts of `pgHead`. Additionally it adds an `@type="autogenerated"` to `pgHead` so it can be identified in SVG more easily.

### CODE REVIEW

File: src/doc.cpp [hunk 1]
```diff
@@ -166,6 +166,7 @@ bool Doc::GenerateHeaderAndFooter()
     // We mark it as generated for not having it written in the output
     pgHead->IsGenerated(true);
     pgHead->GenerateFromMEIHeader(m_header);
+    pgHead->SetType("autogenerated");
```
Comment (lpugin) [inline src/doc.cpp]: Shall we have this in the footer too?

File: src/num.cpp [hunk 2]
```diff
@@ -23,7 +23,7 @@ namespace vrv {
 // Num
 //----------------------------------------------------------------------------
 
-Num::Num() : TextElement("num-"), AttLabelled()
```
Comment (lpugin) [inline src/num.cpp]: Removing label on num will break auto pagination. Why not suggesting to add this to MEI?

Comment (lpugin) [reply src/num.cpp]: Sorry, I did not realized it is there through TextElement.

File: src/doc.cpp (hunk 1 above)
Comment (rettinghaus) [reply src/doc.cpp]: makes sense

**Pull request merged**